frontend/schema/class-schema.php

```diff
@@ -85,6 +105,30 @@ public function generate() {
 			}
 		}
 
+		foreach ( $this->parsed_blocks as $block_type => $blocks ) {
+			/**
+			 * Filter: 'yoast/schema/block-type/<block-type>' - Allows filtering graph output based on all the blocks of a certain type.
+			 *
+			 * @param WP_Block_Parser_Block $blocks  All the blocks of this type.
+			 * @param WPSEO_Schema_Context  $context A value object with context variables.
+			 *
+			 * @api array $graph Our Schema output.
+			 */
+			$graph = apply_filters( 'yoast/schema/block-type/' . $block_type, $graph, $blocks, $this->context );
```

Comment:
The usecase for this filter seems to be solved differently for faq questions now, as it is not being used. Is this filter needed now?

Reply:
Reconsidered, removing it.